consumers/src/unit-test/java/uk/gov/moj/sdt/cmc/consumers/response/ClaimStatusUpdateResponseTest.java

```diff
@@ -0,0 +1,27 @@
+package uk.gov.moj.sdt.cmc.consumers.response;
+
+import com.fasterxml.jackson.core.JsonProcessingException;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import org.junit.jupiter.api.Test;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertNotNull;
+
+public class ClaimStatusUpdateResponseTest {
+
+    @Test
+    void testEnumSerialisation() throws JsonProcessingException {
+        ClaimStatusUpdateResponse response = new ClaimStatusUpdateResponse();
+        response.setProcessingStatus(ProcessingStatus.PROCESSED);
+        String json = new ObjectMapper().writeValueAsString(response);
+        assertNotNull(json);
```

Comment:
Add code to check that the json string contains the expected processingStatus value.  The json string should be something like the one in the deserialisation test below.

Reply:
done